resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -43,6 +43,10 @@ COPY sbin /opt/spark/sbin
 COPY ${img_path}/spark/entrypoint.sh /opt/
 COPY examples /opt/spark/examples
 COPY data /opt/spark/data
+RUN if [ -z "$SPARK_USER" ]; then \
```

Comment:
So this part I _think_ belongs in the entrypoint where you know which user your adding because this is done at image build time where we don't know our deal.

Reply:
Thank you for this catch, I will edit this to see if it solves the `javax.security.auth.login.LoginException`